modules/runner/src/integrationTest/java/org/apache/ignite/internal/sqllogic/Statement.java

```diff
@@ -113,11 +113,14 @@ void execute(ScriptContext ctx) {
                     Assertions.fail("Not expected result at: " + posDesc + ". Statement: " + qry, e);
                 }
             } else {
-                Throwable err = Assertions.assertThrows(Throwable.class, () -> ctx.executeQuery(qry),
+                Throwable err = Assertions.assertThrows(
+                        Throwable.class,
+                        () -> ctx.executeQuery(qry),
                         "Not expected result at: " + posDesc + ". Statement: " + qry + ". Error: " + expected.errorMessage);
```

Comment:
```suggestion
                Throwable err = Assertions.assertThrows(
                        Throwable.class,
                        () -> ctx.executeQuery(qry)
                );
```

Reply:
Why do you want to delete the message?

Reply:
Because this message is irrelevant and will be printed if no exception (or wrong) is thrown.
The message from the exception is validated on the next line.